gradle.properties

```diff
@@ -6,7 +6,6 @@ jdk.version=1.8
 #
 # build
 #
-google-java-format-plugin.version=0.8
```

Comment:
The plugin `id("Y") version "X"` form doesn't allow using variables in the version, so no reason to keep this in properties.